src/main/bash/sdkman-init.sh

```diff
@@ -140,6 +140,9 @@ if [[ ! -f "${SDKMAN_DIR}/var/delay_upgrade" ]]; then
 	touch "${SDKMAN_DIR}/var/delay_upgrade"
 fi
 
+# set version-check
+if [[ -z "$sdkman_version_check" ]]; then sdkman_version_check=true; fi
```

Comment:
We don't need this defensive code as it should be set in the config file.

Reply:
Absolutely. My bad.